docs/src/man/comparisons.md

```diff
@@ -25,25 +27,29 @@ df = pd.DataFrame({'grp': [1, 2, 1, 2, 1, 2],
                    index = list('abcdef'))
 ```
 
-By comparison, this pandas data frame has `a` to `f` as row indices rather than a separate `id` column.
+Because pandas supports multi-index, this example data frame is set up with `a` to `f`
+as row indices rather than a separate `id` column.
 
 ### Accessing data
 
 | Operation                  | pandas                 | DataFrames.jl                      |
-| :------------------------- | :--------------------- | :--------------------------------- |
+|----------------------------|------------------------|------------------------------------|
 | Cell indexing by location  | `df.iloc[1, 1]`        | `df[2, 2]`                         |
 | Row slicing by location    | `df.iloc[1:3]`         | `df[2:3, :]`                       |
 | Column slicing by location | `df.iloc[:, 1:]`       | `df[:, 2:end]`                     |
 | Row indexing by label      | `df.loc['c']`          | `df[findfirst(==('c'), df.id), :]` |
-| Column indexing by label   | `df.loc[:, 'x']`       | `df[:, :x]`                        |
-| Column slicing by label    | `df.loc[:, ['x','z']]` | `df[:, [:x, :z]]`                  |
-|                            | `df.loc[:, 'x':'z']`   | `df[:, Between(:x, :z)]`           |
+| Column indexing by label   | `df.loc[:, 'x']`       | `df[:, [:id, :x]`                  |
+| Column slicing by label    | `df.loc[:, ['x','z']]` | `df[:, [:id, :x, :z]]`             |
+|                            | `df.loc[:, 'x':'z']`   | `select(df, :id, Between(:x, :z))` |
```

Comment:
I think I'd show both syntaxes, with and without `:id` (one per line). Otherwise we don't show how to extract a single column when you don't want labels, which is an essential feature and that users may be surprised not to see.

Reply:
Yeah, I felt this is a little odd as well here. Maybe it's better to break our rule and let go the label in this case. 

I will remove the following note as well.  Saying more about this will probably add more confusion to the reader.
> The DataFrames.jl examples for column indexing include the `id` column so that the result matches pandas' as they appear as row indices there.